sql/core/src/main/scala/org/apache/spark/sql/DataFrameWriter.scala

```diff
@@ -25,13 +25,13 @@ import org.apache.hadoop.fs.Path
 
 import org.apache.spark.annotation.Experimental
 import org.apache.spark.sql.catalyst.TableIdentifier
-import org.apache.spark.sql.catalyst.analysis.UnresolvedRelation
+import org.apache.spark.sql.catalyst.analysis.{Update, OutputMode, Append, UnresolvedRelation}
```

Comment:
(I think it will throws an error for style check and should be as below:)

```scala
import org.apache.spark.sql.catalyst.analysis.{Append, OutputMode, UnresolvedRelation, Update}
```